lib/engine-universal/src/engine.rs

```diff
@@ -95,6 +95,63 @@ impl UniversalEngine {
         binary: &[u8],
         tunables: &dyn Tunables,
     ) -> Result<crate::UniversalExecutable, CompileError> {
+        // Compute the needed instrumentation
+        struct MaxStackCfg;
+        impl finite_wasm::max_stack::Config for MaxStackCfg {
+            fn size_of_value(&self, ty: finite_wasm::wasmparser::ValType) -> u8 {
+                use finite_wasm::wasmparser::ValType;
+                match ty {
+                    ValType::I32 => 4,
+                    ValType::I64 => 8,
+                    ValType::F32 => 4,
+                    ValType::F64 => 8,
+                    ValType::V128 => 16,
+                    ValType::FuncRef => 8,
+                    ValType::ExternRef => 8, // TODO: is this correct?
```

Comment:
I believe this is entirely unsupported in wasmer at this time (it is an extension that I believe hansn't been merged to the base spec yet). Lets just `todo!()` this.

Reply:
Turns out we actually have tests that use extern refs, and we also store extern refs in a GPR so size 8 seems reasonable